src/test/java/uk/gov/pay/directdebit/payments/services/PaymentServiceTest.java

```diff
@@ -140,15 +138,13 @@ public void shouldCreateACollectPaymentResponseFromAValidTransaction() {
     }
     @Test
     public void shouldCreateATransactionResponseWithLinksFromAValidTransaction() {
-        when(mockedTokenService.generateNewTokenFor(mandateFixture.toEntity())).thenReturn(new Token("token", mandateFixture.getId()));
-        PaymentResponse paymentResponse = service
-                .createPaymentResponseWithAllLinks(paymentFixture.toEntity(),
-                        gatewayAccountFixture.getExternalId(), mockedUriInfo);
+        CollectPaymentResponse paymentResponse = service
+                .createPaymentResponseWithAllLinks(paymentFixture.toEntity()
+                );
```

Comment:
the above three lines can be formatted better into one line: `var paymentResponse = service.createPaymentResponseWithAllLinks(paymentFixture.toEntity());`